src/middleware/csrf.js

```diff
@@ -6,6 +6,13 @@ const {
 	generateToken,
 	csrfSynchronisedProtection,
 } = csrfSync({
+	getTokenFromRequest: (req) => {
+		if (req.headers['x-csrf-token']) {
+			return req.headers['x-csrf-token'];
+		} else if (req.query) {
+			return req.query._csrf;
+		}
+	},
```

Comment:
Once we update the form to use hidden fields, we can revert this commit?

Reply:
Once updated with hidden fields, it would change to:

    else if (req.body) {
        return req.body['CSRFToken'];
    }

Where CSRFToken is the name of the field.

Reply:
Thank you :smile:

Reply:
Is the property `CSRFToken` significant, or could it be anything (i.e. `csrf_token`, etc.?)

Reply:
In the example I just gave, it corresponds to the `name` of the hidden field on the form.